CONTRIBUTING.md

```diff
@@ -68,7 +68,10 @@ contributing to the Open Application Model specification. All issue types follow
 Like any good open source project, we use Pull Requests (PRs) to track code changes. To submit a change to the specification:
 
 1. Fork the repo, modify the specification to address the issue.
-1. Submit a pull request.
+2. Pick an open issue from the [issue list](https://github.com/oam-dev/spec/issues) and claim it in the comments. After approval fix the issue and send us a pull request (PR).
+3. Or you can create a new issue. A community member will get back to you and, if approved, you can fix the issue and send a pull request.
```

Comment:
Is it necessary to make a PR before the approval?

Reply:
@zzxwill 3rd point said that `if approved` then send PR.